src/revolve/main.py

```
@@ -0,0 +1,133 @@
+from langgraph.graph import StateGraph, START, END
+
+from revolve.data_types import State
+from revolve.db import get_adapter
+
+from langgraph.constants import Send
+from revolve.utils_git import *
```

Ruff (0.11.9)

`from revolve.utils_git import *` used; unable to detect undefined names

(F403)